cmd/minikube/cmd/logs.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/cluster"
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+// logsCmd represents the logs command
+var logsCmd = &cobra.Command{
+	Use:   "logs",
+	Short: "Gets the logs of a local kubernetes cluster.",
+	Long:  `Gets the logs of a local kubernetes cluster.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+		defer api.Close()
+		s, err := cluster.GetHostLogs(api)
+		if err != nil {
+			log.Println("Error getting machine logs:", err)
+			os.Exit(1)
+		}
+		fmt.Fprintln(os.Stdout, s)
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(logsCmd)
```

Comment:
I think you should attach the `--v` flag here. See how to do that in the cobra documentation